iBudget/utils/aws_helper.py

```diff
@@ -0,0 +1,75 @@
+"""
+File Handler
+Provides methods for upload and download processes of the object to Amazon S3 service
+"""
+import boto3
+
+from ibudget.settings import AWS_ACCESS_KEY_ID, AWS_SECRET_ACCESS_KEY, \
+    AWS_STORAGE_BUCKET_NAME
+
+
+class AwsService:
+    """the class which handles actions with files on AWS
+    """
+    boto_resource = boto3.resource(
+        's3',
+        aws_access_key_id=AWS_ACCESS_KEY_ID,
+        aws_secret_access_key=AWS_SECRET_ACCESS_KEY
+    )
+    boto_bucket = boto_resource.Bucket(AWS_STORAGE_BUCKET_NAME)
+    boto_client = boto3.client(
+        's3',
+        aws_access_key_id=AWS_ACCESS_KEY_ID,
+        aws_secret_access_key=AWS_SECRET_ACCESS_KEY,
+    )
+
+    @classmethod
```

Comment:
Why all those methods are static?

Reply:
I chose to make the methods class methods to be able to call them from the class, otherwise in order to call this method in a view we would have to create an instance of the class AwsService and call it then which seemed to be incorrect for me.

Should I rewrite the methods and make them instance methods?

Reply:
Yes, it is better to have them as instance methods.